src/main/java/mate/academy/controller/BookController.java

```diff
@@ -33,4 +37,16 @@ public BookDto findBookById(@PathVariable Long id) {
     public BookDto createBook(@RequestBody CreateBookRequestDto bookRequestDto) {
         return bookService.save(bookRequestDto);
     }
+
+    @ResponseStatus(HttpStatus.NO_CONTENT)
+    @DeleteMapping("/{id}")
+    void delete(@PathVariable Long id) {
```

Comment:
never return void, consider returning ResponseEntity<?> with proper status code, and create dto for this, that shows delete was successfully finished or not

Reply:
not fixed